### PR TITLE
Add license for BIP75

### PR DESCRIPTION
**Pull request by techguy613**
Add CC-BY license for BIP75

### CODE REVIEW

File: bip-0075.mediawiki [hunk 1]
```diff
@@ -21,6 +21,12 @@ This BIP is an extension to BIP 70 that provides two enhancements to the existin
 The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "SHOULD NOT", "RECOMMENDED", "MAY", and 
 "OPTIONAL" in this document are to be interpreted as described in RFC 2119.
 
+==Copyright==
+
+<img src="https://licensebuttons.net/l/by/3.0/88x31.png">
+
+This BIP is licensed under CC BY.
```
Comment (luke-jr) [inline bip-0075.mediawiki]: The version should be explicit (note that 4.0 is the only acceptable version).

**Comment by luke-jr** [comment]
@justinwnewton @voisine @jmacwhyte @jonathancross  Are you all okay with CC-BY-4.0?

**Comment by jmacwhyte** [comment]
Yes, sounds good to me.

**Comment by justinwnewton** [comment]
Yes.

**Comment by jonathancross** [comment]
Sorry for the late reply.

> Are you all okay with CC-BY-4.0?

Sounds good.

File: bip-0075.mediawiki (hunk 1 above)
Comment (SamiQu) [reply bip-0075.mediawiki]: > The version should be explicit (note that 4.0 is the only acceptable version).

**Comment by SamiQu** [comment]
> Yes, sounds good to me.

**Pull request merged**